t/14-grutasks.t

```diff
@@ -671,7 +671,6 @@ subtest 'git clone' => sub {
     is $res->{result}, 'Job successfully executed', 'minion job result indicates success';
     like $mocked_git_calls[3], qr'git -C /etc/ fetch origin master', 'fetch origin master for /etc/';
     like $mocked_git_calls[4], qr'reset --hard origin/master', 'reset origin/master for /etc/';
-    like $mocked_git_calls[8], qr'git -C /root/ fetch origin foobranch:foobranch', 'fetch non-default branch';
```

Comment:
Why did you remove this test? Don't we still want to test that command? Just that it changed?

Reply:
We do not use any more. there is not such call with refspec in this format. Unless you want to test that it fails!

Reply:
The call is still there. I can see it if I checkout your PR.
It's just not `git -C /root/ fetch origin foobranch:foobranch` anymore, but `git -C /root/ fetch origin foobranch`.

Reply:
bring it back on f8e3adb2b. I had remove it as I dont think is much different from the test 2 lines above.

Reply:
Sure, but it's a different use case.
And as we saw in [the other pull request](https://github.com/os-autoinst/openQA/pull/5863#discussion_r1725013852), we actually would have produced an error in production because we didn't test all elements in that array.
Removing an existing test is rarely a good idea.